main.cpp

```diff
@@ -0,0 +1,32 @@
+
+#include <QApplication>
+#include <QHBoxLayout>
+#include <QVBoxLayout>
+#include <QSlider>
+#include <QSpinBox>
+
+int main(int argc, char *argv[])
+{
+    QApplication app(argc, argv);
+
+    QWidget *window = new QWidget;
+    window->setWindowTitle("Enter Your Age");
+
+    QSpinBox *spinBox = new QSpinBox;
+    QSlider *slider = new QSlider(Qt::Horizontal);
+    spinBox->setRange(0, 130);
+    slider->setRange(0, 130);
+
+    QObject::connect(spinBox, SIGNAL(valueChanged(int)), slider, SLOT(setValue(int)));
+    QObject::connect(slider, SIGNAL(valueChanged(int)), spinBox, SLOT(setValue(int)));
+    spinBox->setValue(35);
+
+    QHBoxLayout *layout = new QHBoxLayout;
+    layout->addWidget(spinBox);
+    layout->addWidget(slider);
+    window->setLayout(layout);
+
+    window->show();
+
+    return app.exec();
```

Comment:
`exec()` is a static function https://doc.qt.io/qt-6/qapplication.html#exec
Consider `QApplication::exec()`